src/styles/modifiers.scss

```
@@ -0,0 +1,75 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+  border: 1px solid $main-border-color;
+  border-radius: 5px;
+  align-items: center;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__name {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $main-color;
+    margin: 0;
+  }
+
+  &__code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $secondary-color;
+    margin: 4px auto 16px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    color: $secondary-color;
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    margin-bottom: 16px;
+  }
+
+  &__price-value {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    color: $main-color;
+  }
```

Use scss nesting everywhere
```suggestion
  &__price {
    display: flex;
    justify-content: space-between;
    color: $secondary-color;
    font-size: 12px;
    font-weight: 400;
    line-height: 18px;
    margin-bottom: 16px;
    
    &-value {
      // write styles here
    }
  }
```